sway/input/cursor.c

```diff
@@ -429,6 +429,9 @@ static void dispatch_cursor_button_floating(struct sway_cursor *cursor,
 		struct sway_container *cont) {
 	struct sway_seat *seat = cursor->seat;
 
+	seat_set_focus(seat, cont);
+	seat_pointer_notify_button(seat, time_msec, button, state);
```

Comment:
I don't think we should send a button event here. If you want to move a window and you drag over a button, it'll click the button.

Reply:
Ah, good catch.